README.md

```diff
@@ -23,16 +23,16 @@ Checkout:
 http://svn.apache.org/repos/asf/karaf/site/trunk/ karaf-site
 ```
 
-Run Build:
+## Development
 
-```
-mvn clean install
-```
+Installation of dev environment, follow official instructions:
+
+https://jekyllrb.com/docs/
 
-## Publish
+## Run
 
-To publish the local website to the production location (https://karaf.apache.org), you have to use:
+To start Jekyll local server, just run:
 
 ```
-mvn install scm-publish:publish-scm
+bundle exec jekyll serve
```

Comment:
It's good to have the "server" mode, but generate mode and publish to scm should stay there.

Reply:
Ok, for deployment, I kept the pom.xml with the war packaging and the scm-publish.
I added more information in the README.